src/main/scala/exercise4/autozion/package.scala

```diff
@@ -0,0 +1,97 @@
+package exercise4
+
+import zio.blocking.{Blocking, effectBlocking}
+import zio.clock.Clock
+import zio.console._
+import zio.duration.{Duration, durationLong}
+import zio.random.{Random, nextIntBounded, nextLongBetween, nextPrintableChar, nextString, nextUUID}
+import zio.{Has, RIO, Schedule, ZIO}
+import java.io.IOException
+
+
+package object autozion {
+  type MyEnv = Has[News] with Has[JobBoard] with Has[CompletedJobsHub] with Random with Clock with Blocking with Console
+  type ElderEnv = Has[JobBoard] with Random with Clock
+  type WorkerEnv = Has[JobBoard] with Has[CompletedJobsHub] with Clock with Blocking  with Console with Random
+  type OverseerEnv = Has[News] with Has[CompletedJobsHub] with Clock
+  type PraiserEnv = Has[News] with Has[CompletedJobsHub] with Clock
+  type ReporterEnv = Has[News] with Clock with Console
```

Comment:
I'm not a fan of adding so many type aliases to the package. 

If I am interested in what services, say, the worker `work`uses, I have to navigate here and check out what this type is. And what if the worker had 2 methods? Would I define `WorkerEnv1` and `WorkerEnv2` here? doesn't seem right to me